docs/annexes/annex-3/annex-3.01-pid-rulebook.md

```diff
@@ -480,7 +522,111 @@ If other CDDL representation types will be used in future versions of
 this document, the corresponding rules for encoding them with JSON will
 be added here.
 
-##### 2.4.3.2 Further stipulations
+
+##### 2.5.3.2 Data Element Identifer to Claim Mapping
+
+SD-JWT encoded PID attestations SHALL use claim names registered in the
+JSON Web Token Claims Registry \[IANA-JWT-Claims\]. Table 2 below maps
+the data element identifiers from Table 1 above to the corresponding
+claim names. 
+
+Note that a hierarchical claim name structure can be used in SD-JWT VC
+encoded PID attestations as SD-JWT allows for individual selective
+disclosure of objects and their properties. A hierarchical claim name
+structure is indicated by the notation `parent.child` in Table 3.
+
+Data element identifiers not listed in Table 3 are used as defined above.
+
+| Data Element Identifier | Claim Name | Note |
```

Comment:
Would it make sense to have next to each register claim a reference  ?
For instance, 

| **Attribute identifier** | **Claim Name**          | **Note**                                                                      | **Reference**                                                                                                                                    |
|--------------------------|-------------------------|-------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
| family_name              | family_name             |                                                                               | [OpenID Connect Core 1.0, Section 5.1](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims)                                     |
| given_name               | given_name              |                                                                               | [OpenID Connect Core 1.0, Section 5.1](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims)                                     |
| birth_date               | birthdate               |                                                                               | [OpenID Connect Core 1.0, Section 5.1](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims)                                     |
|                          |                         |                                                                               |                                                                                                                                                  |
| age_over_18              | age_equal_or_over.18    | The `18` property name is a string.                                           | PID rulebook                                                                                                                                     |
| age_over_NN              | age_equal_or_over.NN    | The NN property name is a string.                                             | PID rulebook                                                                                                                                     |
| age_in_years             | age_in_years            |                                                                               | PID rulebook                                                                                                                                     |
| age_birth_year           | age_birth_year          |                                                                               | PID rulebook                                                                                                                                     |
|                          |                         |                                                                               |                                                                                                                                                  |
| family_name_birth        | birth_family_name       |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| given_name_birth         | birth_given_name        |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| birth_place              | place_of_birth.locality |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| birth_country            | place_of_birth.country  |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| birth_state              | place_of_birth.region   |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| birth_city               | place_of_birth.locality |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| resident_address         | address.formatted       |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| resident_country         | address.country         |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| resident_state           | address.region          |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| resident_city            | address.locality        |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| resident_postal_code     | address.postal_code     |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| resident_street          | address.street_address  |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| resident_house_number    | address.house_number    |                                                                               | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| gender                   | gender                  | Data type mismatch - JWT claims use `male`, `female` and custom text values.* | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| nationality              | nationalities           | Defined as an array, here containing only one entry.                          | [OpenID Connect for Identity Assurance Claims Registration 1.0, Section 4](https://openid.net/specs/openid-connect-4-ida-claims-1_0.html#claims) |
| issuance_date            | iat                     |                                                                               | [RFC7519](https://www.rfc-editor.org/rfc/rfc7519.html)                                                                                           |
| expiry_date              | exp                     |                                                                               | [RFC7519](https://www.rfc-editor.org/rfc/rfc7519.html)                                                                                           |
| issuing_authority        | issuing_authority       |                                                                               | PID rulebook                                                                                                                                     |
| document_number          | document_number         |                                                                               | PID rulebook                                                                                                                                     |
| administrative_number    | administrative_number   |                                                                               | PID rulebook                                                                                                                                     |
| issuing_country          | issuing_country         |                                                                               | PID rulebook                                                                                                                                     |
| issuing_jurisdiction     | issuing_jurisdiction    |                                                                               | PID rulebook                                                                                                                                     |